rosbag2_compression/src/rosbag2_compression/sequential_compression_writer.cpp

```diff
@@ -318,7 +318,7 @@ void SequentialCompressionWriter::compress_file(
       }
     }
 
-    if (!rcpputils::fs::remove(file_relative_to_pwd)) {
+    if (!std::filesystem::remove(file_relative_to_pwd)) {
       ROSBAG2_COMPRESSION_LOG_ERROR_STREAM(
         "Failed to remove original pre-compressed bag file: \"" <<
           file_relative_to_pwd.string() << "\". This should never happen - but execution " <<
```

Comment:
Need to use a non-throwing overload of the `fs::remove(..)` here. 
```suggestion
    if (std::error_code ec; !fs::remove(file_relative_to_pwd, ec)) {
      ROSBAG2_COMPRESSION_LOG_ERROR_STREAM(
        "Failed to remove original pre-compressed bag file: \"" <<
          file_relative_to_pwd.string() << "\". " << ec.message() <<
          ". This should never happen - but execution " <<
```

Reply:
done